scripts/update-jazzy-docs.sh

```diff
@@ -0,0 +1,22 @@
+. scripts/common.sh
+
+function updateDocs() {
+  WORKSPACE=$1
+  SCHEME=$2
+  CONFIGURATION=$3
+  SIMULATOR=$4
+  MODULE=$5
+
+  ensure_simulator_available "${SIMULATOR}"
+  SIMULATOR_GUID=`simulator_ids "${SIMULATOR}"`
+  DESTINATION='id='$SIMULATOR_GUID''
+
+  set -x
+  killall Simulator || true
+  jazzy --config .jazzy.yml -m "${MODULE}" -x -workspace,"${WORKSPACE}",-scheme,"${SCHEME}",-configuration,"${CONFIGURATION}",-derivedDataPath,"${BUILD_DIRECTORY}",-destination,"$DESTINATION"
+  set +x
+}
+
+./scripts/update-jazzy-config.rb
```

Comment:
If this script is incorporated into the release process (via `release-prep.sh`), this line should be removed (or removed from `release-prep.sh`).